src/main/java/gov/nj/innovation/customAwsIdp/awscdk/AwsIdpCdkStack.java

```diff
@@ -0,0 +1,163 @@
+package gov.nj.innovation.customAwsIdp.awscdk;
+
+import software.amazon.awscdk.CfnOutput;
+import software.amazon.awscdk.Duration;
+import software.amazon.awscdk.aws_apigatewayv2_authorizers.HttpJwtAuthorizer;
+import software.amazon.awscdk.aws_apigatewayv2_integrations.HttpLambdaIntegration;
+import software.amazon.awscdk.services.apigatewayv2.AddRoutesOptions;
+import software.amazon.awscdk.services.apigatewayv2.CorsHttpMethod;
+import software.amazon.awscdk.services.apigatewayv2.CorsPreflightOptions;
+import software.amazon.awscdk.services.apigatewayv2.HttpApi;
+import software.amazon.awscdk.services.apigatewayv2.PayloadFormatVersion;
+import software.amazon.awscdk.services.iam.Effect;
+import software.amazon.awscdk.services.iam.PolicyStatement;
+import software.amazon.awscdk.services.lambda.Code;
+import software.amazon.awscdk.services.lambda.Function;
+import software.amazon.awscdk.services.lambda.Runtime;
+import software.amazon.awscdk.services.logs.LogGroup;
+import software.constructs.Construct;
+import software.amazon.awscdk.Stack;
+import software.amazon.awscdk.StackProps;
+
+import java.text.MessageFormat;
+import java.util.List;
+import java.util.stream.Stream;
+
+import static gov.nj.innovation.customAwsIdp.util.Constants.AWS_ACCOUNT_ID;
+import static gov.nj.innovation.customAwsIdp.util.Constants.AWS_REGION;
+import static gov.nj.innovation.customAwsIdp.util.Constants.COGNITO_APP_CLIENT_ID;
+import static gov.nj.innovation.customAwsIdp.util.Constants.COGNITO_USER_POOL;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_CRT_COEFFICIENT_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_PRIME_EXPONENT_P_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_PRIME_EXPONENT_Q_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_PRIME_P_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_PRIME_Q_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.KEY_PRIVATE_EXPONENT_NAME;
+import static gov.nj.innovation.customAwsIdp.util.Constants.PATH_PARAMETER_GROUP_NAME;
+
+/**
+ * Stack for deploying infrastructure using the AWS CDK.
+ * <p>
+ * This stack currently includes:
+ * <ul>
+ *     <li>An HTTP API</li>
+ *     <li>A JWT Authorizer</li>
+ *     <li>A custom log group</li>
+ *     <li>The Lambda function</li>
+ *     <li>An HTTP Lambda Integration, to connect the APIGateway Route to the HTTP API</li>
+ *     <li>The APIGateway Authorizer/Lambda route</li>
+ *     <li>All the required permissions for the Lambda role</li>
+ * </ul>
+ *
+ * @author Case Walker (case@innovation.nj.gov)
+ */
+public class AwsIdpCdkStack extends Stack {
```

Comment:
This file tracks very closely with the CloudFormation Stack, so I tried to make it as close to identical to the old Serverless stack for this project as possible.